lib/entities/quote/DutchQuote.ts

```diff
@@ -358,15 +358,19 @@ export abstract class DutchQuote<T extends DutchQuoteRequest> implements IQuote
   }
 
   // Returns the number of gas units extra required to execute this quote through UniswapX
-  static getGasAdjustment(classicQuote: ClassicQuote): BigNumber {
+  static getGasAdjustment(classicQuote: ClassicQuote, adjustmentBps?: number): BigNumber {
```

Comment:
Another option.. if you default it to 0, then you can remove the conditional.  Lines 370-373 becomes
```
result.mul(BPS - adjustmentBps).div(BPS);
```
```suggestion
  static getGasAdjustment(classicQuote: ClassicQuote, adjustmentBps?: number = 0): BigNumber {
```